docs/index.rst

```diff
@@ -44,6 +47,8 @@ Tempo is very easy to use:
     phone_accel_tsdf = TSDF(phone_accel_df, ts_col="event_ts", partition_cols = ["User"])
     display(phone_accel_tsdf)
 
+.. _direct-git-install:
+
```

Comment:
I'm not sure why this line appears - can we remove it? Here is what I'm seeing as raw text in the index.rest and I think we could re-word it. I would suggest something like - time series on Spark & Photon is the highly performant for historical analyses. We are simplifying all the common functions to make development more easy.

[:ref:`Much of the Spark ecosystem already uses <who-uses>`](https://github.com/databrickslabs/tempo/blob/enrichDocHostingInfo/docs/index.rst#id1) tempo for time-series analysis and you should too.

Reply:
Yes of course we can rephrase the text content in the docs. I will make this change (rephrase the text with what you mentioned and I will also remove that reference that you see there `:ref:`). 

In fact once the docs workflow is setup by Tristan and we have it live. It would be helpful to have the team's feedback on all of the pages.

Reply:
The suggested change has been pushed.